src/partials/cart.hbs

```diff
@@ -0,0 +1,63 @@
+<div class="wrapper">
+  <!--Login Section-->
+  <div class="cart-section">
+    <div class="cart_header">
+      <h3>My Cart</h3>
+      <span>(1 item)</span>
+    </div>
+    <div class="cart_contents">
+      <table class="cart_contents-table">
+        <tr class="cart_contents-row">
+          <td>
+            <div class="cart_content-img-container">
+              <div>image</div>
+            </div>
+          </td>
+          <td>
+            <div class="cart_content-descp">
+              Item Name
```

Comment:
Create a JSON file with all these text entry which will act like i18n so you can localise it later . It is not recommended to keep any string value hard-coded in template

Reply:
will be changing to i18n